catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java

```diff
@@ -120,16 +129,18 @@ private void executeRemoveAllFromCache() throws Exception {
 
     long end = System.currentTimeMillis();
 
-    String info = String.format("Cache cleared in %3.3f seconds%n", (end - start) / MS_PER_SECOND);
+    String info = String.format("Cache cleared in %3.3f seconds", (end - start) / MS_PER_SECOND);
 
     LOGGER.info(info);
-    LOGGER.info("Cache cleared by catalog:removeAll with --cache option");
+    LOGGER.info("Cache cleared using the \"{} --cache\" command", COMMAND);
 
     console.println();
-    console.print(info);
+    console.println(info);
```

Comment:
Updated output:
```
admin@root()> catalog:removeall --cache
WARNING: This will permanently remove all records from the cache. Do you want to proceed? (yes/no): yes

Cache cleared in 0.007 seconds
admin@root()> log:tail
08:55:34,836 | INFO  | emoveall --cache | ce.ddf.commands.catalog.RemoveAllCommand  131 | atalog-core-commands | Cache cleared in 0.007 seconds
08:55:34,836 | INFO  | emoveall --cache | ce.ddf.commands.catalog.RemoveAllCommand  132 | atalog-core-commands | Cache cleared using the "catalog:removeall --cache" command
```